<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>431</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ГИС данные</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string>Поле карты</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Состояние системы</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Камера</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>586</x>
        <y>0</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Контроль шасси</string>
      </property>
      <widget class="QPushButton" name="forwardButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>22</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Вперед</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rightUpButton">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>50</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ПравыйП</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>128</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
      <widget class="QPushButton" name="leftUpButton">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>50</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ЛевыйП</string>
       </property>
      </widget>
      <widget class="QPushButton" name="strafeRButton">
       <property name="geometry">
        <rect>
         <x>103</x>
         <y>75</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ПТ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="strafeLButton">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>75</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ЛТ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rightDwButton">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>100</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ПравыйН</string>
       </property>
      </widget>
      <widget class="QPushButton" name="leftDwButton">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>100</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ЛевыйН</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>586</x>
        <y>163</y>
        <width>181</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Статус контроллера</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
